Automattic/wp-calypso
# Site Settings: Display upgrade nudge using siteHasFeature for CDN

#### Changes proposed in this Pull Request

* This PR replaces various plan checks in `Site Settings > Performance > CDN` with a new siteHasFeature checks by adding CDN constants. 

#### Testing instructions


- Apply D81072-code your sandbox and sandbox public api.
- Visit Site Settings > Performance
- Look for upgrade nudges under the CDN section.  Visit this section by selecting sites with various plans. 
- Make sure you see something similar as below.

**Free or Personal plan should display upgrades nudges:**

<img width="738" alt="Screenshot 2022-05-19 at 12 11 14 PM" src="https://user-images.githubusercontent.com/86406124/169232965-87b434d9-b572-4844-9947-b3df5a4133bf.png">

**Premium plan site should display upgrade nudge for Jetpack CDN only:**

<img width="741" alt="Screenshot 2022-05-19 at 12 13 43 PM" src="https://user-images.githubusercontent.com/86406124/169232996-b1554298-d75b-48a4-8df7-e05e425205b5.png">

**Pro and Higher plans should not display any upgrade nudge:**

<img width="735" alt="Screenshot 2022-05-19 at 12 14 28 PM" src="https://user-images.githubusercontent.com/86406124/169233005-10ca7f01-9ce1-4def-9beb-fe84710c9235.png">



Related to p4TIVU-a66-p2
